src/JavascriptVM/JavascriptVM.tsx

```diff
@@ -39,6 +39,7 @@ import {
   challengeSlice,
   ChallengeStatus,
 } from "../RobotSimulator/Arenas/challengeSlice";
+import { SimBasicSensor } from "@fruk/simulator-core/dist/engine/objects/robot/sensors/SimBasicSensor";
```

Comment:
Why was that import added?

Reply:
That was a mistake, i think VsCode has a habbit of adding these when they're not needed! :)